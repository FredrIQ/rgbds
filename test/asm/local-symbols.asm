SECTION "Test", ROM0[0]

GlobalScope: ; Define a global scope for locals to not error out on that

.label:
.equ EQU 0
.equs EQUS "Hello tester"
.set SET 42
.macro: MACRO
    ;
ENDM
