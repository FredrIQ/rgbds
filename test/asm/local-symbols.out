ERROR: local-symbols.asm(6):
    Cannot create "local" EQU '.equ'
ERROR: local-symbols.asm(7):
    Cannot create "local" EQUS '.equs'
ERROR: local-symbols.asm(8):
    Cannot create "local" SET '.set'
ERROR: local-symbols.asm(11):
    Cannot create "local" macro '.macro'
error: Assembly aborted (4 errors)!
